src/main/java/com/urswolfer/gerrit/client/rest/http/accounts/AccountApiRestClient.java

```diff
@@ -80,6 +81,23 @@ public BinaryResult downloadAvatar(int size) throws RestApiException {
         }
     }
 
+    @Override
+    public void addEmail(EmailInput input) throws RestApiException {
+        String request = getRequestPath() + "/emails/" + Url.encode(input.email);
+
+        // TODO handle the case where no confirmation is specified
+        // This probably requires the creation of an EmailInfo class in the
+        // Google Java API (com.google.gerrit.extensions.api)
+
+        // Perform the request to update the e-mail
+        JsonElement emailResult = gerritRestClient.putRequest(request);
```

Comment:
Minor fix required, the JSON body needs to be included in the PUT request:

        String request = getEmailRequestPath() + Url.encode(input.email);
        String body = gerritRestClient.getGson().toJson(input);
        JsonElement emailResult = gerritRestClient.putRequest(request, body);